app/utils/push_notifications.js

```diff
@@ -63,7 +63,7 @@ const onPushNotification = async (deviceNotification) => {
     let stopLoadingNotification = false;
 
     // mark the app as started as soon as possible
-    if (Platform.OS === 'android' && !app.appStarted) {
+    if (!app.appStarted) {
```

Comment:
What's changed here that we want to do this on iOS as well?

Reply:
that now we need to know if the app was open from a push notification regardless the platform